pkgs/tools/nix/nixci/default.nix

```diff
@@ -1,32 +1,37 @@
 { lib, stdenv
 , rustPlatform
 , fetchCrate
+, fetchFromGitHub
 , libiconv
 , openssl
 , pkg-config
 , Security
-, devour-flake
```

Comment:
Should this package be removed from `pkgs/tools/nix/devour-flake/default.nix`?

Reply:
Sure, done.